src/inspector_agent.cc

```diff
@@ -429,6 +430,33 @@ class ChannelImpl final : public v8_inspector::V8Inspector::Channel {
   std::unique_ptr<v8_inspector::V8InspectorSession> session_;
 };
 
+class RepeatingInspectorTimer {
+ public:
+  RepeatingInspectorTimer(uv_loop_t* loop, double interval,
+                 v8_inspector::V8InspectorClient::TimerCallback callback,
+                 void* data) : timer_(uv_timer_t()), callback_(callback),
```

Comment:
Doesn’t just `timer_()` work as well?

Also, I’d really rather align the arguments (and intializers) vertically for readability here